README.md

```diff
@@ -8,12 +8,12 @@ In calling .yml workflow file:
 ```
 name: PHP syntax check
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
 
   php-syntax-check:
-    uses: dxw/govpress-workflow-php-syntax-check/.github/workflows/php-syntax-check.yml@v0.1
+    uses: dxw/govpress-workflow-php-syntax-check/.github/workflows/php-syntax-check.yml@v1.0
```

Comment:
I'd suggest we just call `@v1` here. Then we can move the `v1` tag forward as we do minor releases, and they'll auto-rollout to the calling repos. See e.g. the way we use tags on the Whippet validate workflow: https://github.com/dxw/govpress-workflow-whippet-validate/tags

Reply:
Fixed @RobjS